stable/fluentd/templates/hpa.yaml

```diff
@@ -7,12 +7,13 @@ metadata:
     app: {{ template "fluentd.name" . }}
     chart: {{ template "fluentd.chart" . }}
     release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}    
+    heritage: {{ .Release.Service }}
 spec:
   scaleTargetRef:
     apiVersion: apps/v1
     kind: StatefulSet
     name: {{ include "fluentd.fullname" . }}
+    serviceName: {{ template "fluentd.serviceName" . }}
```

Comment:
Why don't you do all the logic in one line here? You can have a default in case the value isn't set

Reply:
@hectorj2f That's amended, PTAL